edge_computing/image-base-install/ibi-preparing-for-image-based-install.adoc

```diff
@@ -8,11 +8,11 @@ toc::[]
 To prepare for an image-based installation for {sno} clusters, you must complete the following tasks:
 
 * Create a seed image by using the {lcao}.
-* Verify that all software components meet the required versions. For further information, see "Software prerequisites for an image-based installation and deployment".
+* Verify that all software components meet the required versions. 
```

Comment:
I think this doc needs an additional headline saying:
`Create a seed image by using the Lifecycle Agent`

Also, I'm unsure what `Verify that all software components meet the required versions.` is referencing (I can't find it in this doc)

Reply:
I believe it references [this section](https://79610--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image-base-install/ibi-understanding-image-based-install#ztp-image-based-upgrade-prereqs_ibi-understanding-image-based-install). I'm not sure why the reference was removed but I'll add it back in with a link in Additional resources

Reply:
As for the headline, we have [Generating a seed image with the Lifecycle Agent](https://79610--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image-base-install/ibi-preparing-for-image-based-install#cnf-image-based-upgrade-generate-seed-image_ibi-preparing-image-based-install).